### DISCUSSION TITLE
Return on rent parameter

### DISCUSSION THREAD

**Discussion by ilhanu** [Fair-Squares/fair-squares#232] [General]
One of the main parameters what makes FS unique is the protocol-wide parameter of the return-on-rent (ROR). This only applies to the assets to be onboarded to what it's currently set. 

This is a constant that is configurable and it sets the return and rent for the investors and tenant. 

A scenario where this configurable constant comes in to play is when an asset is finalized, we have to add the rental price for a house.  `sale_price * ROR =  return on rent or rental price`. 

Think setting is as a starting parameter to 3%, not sure if this should be allowed to be changed by democracy, or another governance body should have the right to change it and then leads to an externalDemocracy. 

I think the implementation when changed and there are already assets that have onboarded and have tenant agreements, this should not change agreement. 

Am I missing something ?

**Comment by ndkazu** [comment]
I think the ROR has to be connected to the duration of the asset occupation and the number of occupants: rather than a constant, it should be an equation.
-  If we want to refund 3% of the asset in one year, for example, an equation could be: 

**_ROR = (3% x Asset price) / [12month x minimum of active tenants for the asset]_**

- **_minimum of active tenants for the asset_** is based on average market rent (AmR). The rent should be less than the AmR:

**_minimum of active tenants for the asset = 1 tenant+(3% x Asset price) / [12month x AmR])_**

- The equation above could be used as passing criteria when evaluating an asset newly submitted by a seller: based on its price, can the asset accommodate the **_[minimum number of tenants ]_**?

then

**_ROR = ((3% x Asset price) x AmR) / ((3% x Asset price) + 12months x AmR)_**

The difficulty is getting the right AmR for each asset:
- Should we use a minimum price per meter square x Area?
- should we consider the location?
- etc...

Clearly, the data used to calculate the AmR, would be gathered through an http request by an off-chain worker.
The good thing is that this equation/Algorithm could be used as a first filter before sending the asset to the council for evaluation, besides providing a rent value.
